Makefile.main

```diff
@@ -287,6 +291,21 @@ deploy: install-helm
 	chmod +x $${HELM_DEPLOY}; \
 	$${HELM_DEPLOY} $(HELM_RELEASE) $(HELM_CHART) $(K8S_NAMESPACE) $(K8S_SERVICE_ACCOUNT) $(HELM_ARGS)
 
+install-python-indexer:
+	pip3 install pypi-on-github-indexer
+
+update-python-index: install-python-indexer
+	@if [ -n "$(TRAVIS_TAG)" ]; then \
+		python3 -m pypi_on_github_indexer --signature "$(PYTHON_INDEX_SIGNATURE)" \
```

Comment:
https://github.com/src-d/pypi-on-github-indexer/issues/13

Reply:
Merged and 0.1.1 released in PyPI https://pypi.org/project/pypi-on-github-indexer/